common/internal_model/src/main/scala/weco/catalogue/internal_model/languages/Language.scala

```diff
@@ -1,3 +1,3 @@
 package weco.catalogue.internal_model.languages
 
-case class Language(label: String, id: String)
+case class Language(id: String, label: String)
```

Comment:
I wonder if doing this causes issues in the other transformers? It will still compile cause they are both strings. Did you check if we use Language without named arguments elsewhere?

Reply:
Yup!